### ISSUE TITLE
Wwise Source Plugins load but don't play

### ISSUE THREAD

**Issue by ashtonmills** [enzienaudio/heavy#231]
Hi there.

I've been trying to add PD patches to Wwise using heavy. I have tried a few different times now and it's not playing back to me. 
I made a really bare bones patch to test, just a 2 phasors; carrier and modulator, muliplying them, with a @hv_param for the FM value. 
The patch preview plays correctly in the Heavy browser, and when I set it up as a Wwise plugin, I can select it as  a source on  a new SFX, and I can adjust the parameter, but when I hit play nothing happens. It just stops again immediately. It's doen this with all the patches I have created.
Any ideas on what I can do?

Thanks a lot

Ashton

**Comment by diplojocus** [comment]
hi @ashtonmills are you testing on Wwise `2017.1`? We haven't yet pushed an official release that supports it. However it is working in the `dev` channel which can be accessed via the `hv-uploader` (`-r dev`). See here for more info: https://enzienaudio.com/docs/index.html#03.uploader#getting-started

**Comment by ashtonmills** [comment]
Thanks @diplojocus  yes I'm using Wwise 2017.1  

Would it be easiest for me to use a previous version of wwise?

**Comment by diplojocus** [comment]
Yes, though if you're familiar with using command line tools the `hv-uploader` should be straightforward.

In a terminal you'd execute something like:
```
$ hv-uploader /path/to/patch/dir/ -n patchname -r dev 
```

**Comment by ashtonmills** [comment]
Thanks. Actually I'm not really familiar with this. Is it something I do in Command promt?

**Comment by diplojocus** [comment]
It's possible to install on Windows, but you'll probably have to install python and configure it. I'd say it would be best to try an older version of Wwise and wait for the next heavy release (should be soon).

**Comment by ashtonmills** [comment]
Great thank you

**Comment by ashtonmills** [comment]
And thanks for beign so swift!

**Comment by diplojocus** [comment]
👍

**Comment by ashtonmills** [comment]
You don't know by any chance how to change the version of Wwise that's integrated to my Ue4 project do you? There seems to be a lot of info on how to integrate, but not how to 'disintegrate'!

**Comment by diplojocus** [comment]
Not 100% sure on the Wwise installation process in UE4 but you could try deleting the `Engine/Plugins/Wwise/` directory, and installing the older version... ...I think

**Comment by ashtonmills** [comment]
I'll try that thanks mate
